This would solve a nasty npm problem we have now that `oxlint` was bumped (around peer-dependencies), so that would be great 🙏 

```
❯ npm install
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: osrd@undefined
npm warn Found: oxlint-tsgolint@0.24.0
npm warn node_modules/oxlint-tsgolint
npm warn   dev oxlint-tsgolint@"^0.23.0" from the root project
npm warn   1 more (oxlint)
npm warn
npm warn Could not resolve dependency:
npm warn peerOptional oxlint-tsgolint@">=0.24.0" from oxlint@1.73.0
npm warn node_modules/oxlint
npm warn   dev oxlint@"^1.73.0" from the root project
npm warn   1 more (vite-plugin-checker)
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: osrd@undefined
npm warn Found: oxlint-tsgolint@0.24.0
npm warn node_modules/oxlint-tsgolint
npm warn   dev oxlint-tsgolint@"^0.23.0" from the root project
npm warn   1 more (oxlint)
npm warn
npm warn Could not resolve dependency:
npm warn peerOptional oxlint-tsgolint@">=0.24.0" from oxlint@1.73.0
npm warn node_modules/oxlint
npm warn   dev oxlint@"^1.73.0" from the root project
npm warn   1 more (vite-plugin-checker)
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: osrd@undefined
npm warn Found: oxlint-tsgolint@0.24.0
npm warn node_modules/oxlint-tsgolint
npm warn   dev oxlint-tsgolint@"^0.23.0" from the root project
npm warn   1 more (oxlint)
npm warn
npm warn Could not resolve dependency:
npm warn peerOptional oxlint-tsgolint@">=0.24.0" from oxlint@1.73.0
npm warn node_modules/oxlint
npm warn   dev oxlint@"^1.73.0" from the root project
npm warn   1 more (vite-plugin-checker)

changed 2 packages, and audited 1263 packages in 1s

2 vulnerabilities (1 low, 1 moderate)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
```

--- a/front/package.json
+++ b/front/package.json
@@ -125,7 +125,7 @@
     "openapi-types": "^12.1.3",
     "oxfmt": "^0.58.0",
     "oxlint": "^1.73.0",
-    "oxlint-tsgolint": "^0.23.0",
+    "oxlint-tsgolint": "^0.24.0",
     "pdfjs-dist": "^6.1.200",
     "sass": "^1.101.0",
     "ts-node": "^10.9.2",

--- a/front/src/modules/simulationResult/components/SpaceTimeChartWrapper/helpers/__tests__/canDragHoveredTrain.spec.ts
+++ b/front/src/modules/simulationResult/components/SpaceTimeChartWrapper/helpers/__tests__/canDragHoveredTrain.spec.ts
@@ -77,7 +77,7 @@ describe('canDragHoveredTrain', () => {
       expect(
         canDragHoveredTrain({
           panelSelectionMode: 'compliant',
-          hoveredTrain: train(OCC_1_0, { start_time: { value: 1000 } as never }),
+          hoveredTrain: train(OCC_1_0, { start_time: { value: 1000 } }),
           selectedTrainId: TRAIN_SCHEDULE_1,
         })
       ).toBe(false);

